docs/book/src/developer/tilt.md

```diff
@@ -323,10 +323,10 @@ Custom values for variable substitutions can be set using `kustomize_substitutio
 
 ```yaml
 kustomize_substitutions:
-  NAMESPACE: default
-  KUBERNETES_VERSION: v1.27.1
-  CONTROL_PLANE_MACHINE_COUNT: 1
-  WORKER_MACHINE_COUNT: 3
+  NAMESPACE: "default"
+  KUBERNETES_VERSION: "v1.27.1"
```

Comment:
If I'm not wrong quotes are not required here because yaml would already detect those as strings.

Also what about adding a note just below the example explaining why quotes are required for non-string values?

Reply:
Yes of course quotes around them are not needed but in a conversation with @killianmuldoon , he suggested me to add these quotes for better clarity

Reply:
also should I add a note something like this - "Under the hood the prepare_all() function expects the machine_counts to be string thus we need to make these integers string by wrapping quotes around them" ??

Reply:
I would add something like

`Note: kustomize_substitutions must be of type strings; this can be achieved by adding quotes to values`

Reply:
Hmm... You are right..